### ISSUE TITLE
Option to Disable/Hide Pull Request Description Content

### ISSUE THREAD

**Issue by reynldi** [integrations/slack#1741]
**Is your feature request related to a problem? Please describe.**
<img width="921" alt="image" src="https://github.com/integrations/slack/assets/28092744/96b57359-7a2c-41ce-aba8-d60b8f8648aa">

Showing the PR Description in the Slack Channel will take so many space. It's painful and annoying to scroll up/down just to see the other PR histories.

**Describe the solution you'd like**
Provide an option to hide the Pull Request Description Content to make message bot more slim.

**Describe alternatives you've considered**
Is it possible to make it collapsable ? it would be great if we can make it as collapsable content.

**Additional context**

**Comment by MWhiteFearless** [comment]
I would love this so much, we really don't need to see the PR description and it makes our channel very noisy. For my team, this is the worst part of the app and seems like it would be an easy fix that makes the experience much better. All we want is a link to the PR and the reviewers 🙏 

I'd also love to be able to opt out of sending the "pull request merged" notifications to the channel and just keep them in the PR thread

**Comment by reynldi** [comment]
> I would love this so much, we really don't need to see the PR description and it makes our channel very noisy. For my team, this is the worst part of the app and seems like it would be an easy fix that makes the experience much better. All we want is a link to the PR and the reviewers 🙏 
> 
> 
> 
> I'd also love to be able to opt out of sending the "pull request merged" notifications to the channel and just keep them in the PR thread


Indeed, this is what we need.

**Comment by aagranovExtend** [comment]
Is there an option to disable preview?

**Comment by rachel-gramlich-alma** [comment]
Agreed here! It would be huuuugely helpful to be able to choose whether or not we want the preview to show up!

**Comment by Beta6484** [comment]
This would be perfect! Why hasn't it been implemented yet?

**Comment by akshayrao14** [comment]
+1

**Comment by kennedy-whytech** [comment]
There's also an [alternative](https://docs.github.com/en/organizations/organizing-members-into-teams/managing-scheduled-reminders-for-your-team) to setup reminder on schedule, which will not include any details in the description. It's not real time, but it save some space in the slack channel.
